examples/mongo/mongo_3.rb

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env ruby
+
+require 'yaml'
+
+# Requires Mongo 3.0 command line tools
+# ( uses --json for output, not available prior to 3.0 )
+
+MONGO_HOST           = ENV["MONGO_HOST"] || "127.0.0.1"
+MONGO_PORT           = ENV["MONGO_PORT"] || 27017
+MONGO_USER           = ENV["MONGO_USER"]
+MONGO_PWD            = ENV["MONGO_PASSWORD"]
+MONGO_AUTH_MECHANISM = ENV["MONGO_AUTH_MECHANISM"]
+MONGO_AUTH_DB        = ENV["MONGO_AUTH_DB"]
+
+CMDS                 = %w{mongotop mongostat}
```

Comment:
Does this assume that mongotop/mongostat are in $PATH?  I don't know that that's true for everyone, many people use custom installs and scripts and ignore path entirely.  May need a configurable MONGO_PATH or something like that.